README.md

```diff
@@ -26,7 +26,7 @@ actionLink: /tryit/
   text-align: center;
 }
 
-.githubbutton {
+.roundedbutton {
   position: relative;
   display: inline-block;
   margin: 0 2em;
```

Comment:
I'll add a margin-top of 1em so on mobile we'll have a margin between the two buttons.

Reply:
margin-top attribute added